README.md

```diff
@@ -5,4 +5,26 @@
 
 # Google Container Tools for IntelliJ
 
-In development
+This plugin integrates continuous development on [Kubernetes](https://www.kubernetes.io) using [Skaffold](https://skaffold.dev) into [Jetbrains family of IDEs](https://www.jetbrains.com/products.html), including IntelliJ (both Community and Ultimate editions), GoLand, PyCharm and WebStorm.
```

Comment:
I'm leaning towards not mentioning Skaffold so soon in the intro paragraph. I'm fine mentioning it at the feature level to say that we achieve deployment / continuous development using Skaffold. How about something like:

> This plugin adds support for Kubernetes development into the JetBrains family of IDEs, including ...

Reply:
> including IntelliJ (both Community and Ultimate editions), GoLand, PyCharm and WebStorm.

..PyCharm, WebStorm (and others). ?

Reply:
Agree, let's skip Skaffold in the initial description, we are going to add more infrastructure to hide it behind the scenes anyway with generated resources and configuration files.

Reply:
done.